config/foreman-proxy-content.migrations/220127140752-enable-pulp-content-cache.rb

```diff
@@ -0,0 +1,3 @@
+if answers['foreman_proxy_content'].is_a?(Hash)
```

Comment:
If you write it more specific you won't need a change in the answers file. This would keep the default as built into the module. It only changes `false` to `true`.
```suggestion
if answers['foreman_proxy_content'].is_a?(Hash) && answers['foreman_proxy_content']['pulpcore_cache_enabled'] == false
```

Reply:
so its safe to assume that it would be false and not nil?

Reply:
Good question so I had to stop and think for a moment. `nil` as a value means `undef` in Puppet. Not having the key means its default has yet to be determined. So it can also be:
```suggestion
if answers['foreman_proxy_content'].is_a?(Hash) && answers['foreman_proxy_content'].key?('pulpcore_cache_enabled')
```

Perhaps that's even cleaner.